build/build.sh

```diff
@@ -19,5 +22,10 @@ rm dist/cli.js
 # types
 cp types/types.d.ts dist/types.d.ts
 
+# unplugin
+yarn --cwd ./integration/unplugin build
+cp ./integration/unplugin/dist/* ./dist
+
```

Comment:
For Windows, I realize `chmod a-x dist/*.*[jt]s` would be helpful.

Also, what is this new `dist/chunk-UZRN3RFA.mjs` file? Presumably it's not desired?

Reply:
For the esm build, tsup splits the common chunks into a separate file by default. As all the bundler entrypoints `/esbuild`, `/rollup` etc. import the central `civetUnplugin` it splits that into this chunk and imports it in them. Mostly helps keep the bundle size smaller as its not duplicated

Reply:
Ah, that is nice. Can we choose a better name for that file? Given that there's only one, maybe a custom name rule would work?

Reply:
We can choose a better name for it https://esbuild.github.io/api/#chunk-names, but i'm not sure if we should. There currently only exists one of these, but might be more than that in the future. I would suggest keeping it as is, or just turning off splitting

Reply:
I think it would be nice to rename it for now; we can always change it if we need multiple chunks in the future (seems unlikely for the unplugin?). Maybe `unplugin-shared`?